openlibrary/templates/books/custom_carousel.html

```diff
@@ -32,7 +32,11 @@
   $ work_key = book.key if book.key.startswith('/work') else book.get('work_key')
   $if lazy:
     $ img_attr = 'data-lazy'
+    $ data_url = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkYAAAAAYAAjCB0C8AAAAASUVORK5CYII='
+    $ src_attr = 'src'
   $else:
+    $ data_url = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkYAAAAAYAAjCB0C8AAAAASUVORK5CYII='
```

Comment:
@jdlrobson If the book has a cover_url **and** it's lazy, it will load with a data-lazy `img_attr`, but if the book is not lazy but has a cover_url, it will load with a src `img_attr`. So here, to solve this issue, I have added a src attribute to lazy books, which points to a 1x1 png.
But as we need to define an else statement for the same and we can't use two src attributes for one image, I defined a `data-src` attr, which can basically point to anything.

May be I am wrong :(